product_docs/docs/pem/9/certificates/index.mdx

```diff
@@ -0,0 +1,312 @@
+---
+title: "Managing SSL certificates"
+navTitle: "Managing SSL certificates"
+legacyRedirectsGenerated:
+  # This list is generated by a script. If you need add entries, use the `legacyRedirects` key.
+  - "/edb-docs/d/edb-postgres-enterprise-manager/user-guides/administrators-guide/8.0/managing_certificates.html"
+# This file is moved from pem_admin/04_managing_certificates
+redirects:
+- /pem/latest/managing_certificates/
+- /pem/latest/pem_admin/04_managing_certificates/
+navigation:
+- replacing_ssl_certificates
+- regenerating_agent_certificates
+---
+
+PEM uses self-signed SSL certificates:
+
+- Webserver(HTTPD)
+- PEM agent and backend database server connection
+
+## Webserver certificates
+
+We recommend an additional layer of SSL security for the web application.
+
+PEM uses the self-signed SSL certificates for the Apache/HTTPD server. The self-signed SSL certificates and keys for the Apache/HTTPD server get generated during the PEM installation.
+
+If you want to use your own SSL certificate for PEM, you must update the Apache configuration file.
+
+On Linux, you need to open the Apache HTTPD configuration file (`edb-ssl-pem.conf`) in a text editor and be a user with write permission on the file. You must also change the server name and file names in the configuration file to match your certificate files.
+
+Update these two SSL directives in the PEM VirtualHost section:
+
+-   `SSLCertificateFile` is your DigiCert certificate file (for example, your_domain_name.crt).
+-   `SSLCertificateKeyFile` is the .key file generated when you created the CSR (for example, your_private.key).
+
+For example, make the following updates:
+
+```shell
+SSLEngine on
+SSLCertificateFile /path/to/your_domain_name.crt
+SSLCertificateKeyFile /path/to/your_private.key
+```
+
+To increase the security, you can replace the HTTPD self-signed SSL certificates with the trusted third-party CA signed certificates in PEM. For more information, see [Replacing HTTPD self-signed SSL certificates](https://www.enterprisedb.com/postgres-tutorials/how-replacing-httpd-self-signed-ssl-certificates-trusted-ca-signed-certificates).
+
+## PEM backend database server and agent connection certificates
+
+PEM, by default, supports secured SSL/TLS connections and acts as its own certificate authority (CA) to generate certificates and keys for the PEM server and agent. The self-signed SSL certificates and keys for the PEM server and agent get generated during the PEM installation. These certificates and keys encrypt the connections between the server and the agent. 
```

Comment:
```suggestion
PEM, by default, implements secured SSL/TLS connections between PEM agents and the backend database, and acts as its own certificate authority (CA) to generate certificates and keys for the PEM server and agent. The self-signed SSL certificates and keys for the PEM server and agent are generated during PEM installation. These certificates and keys encrypt the connection from agent to server. In addition, PEM Agents are authenticated using their certificate rather than a password.
```